examples/react-server-components/lib/use-data.js

```diff
@@ -0,0 +1,55 @@
+const cache = {}
```

Comment:
😱

Could we please fast track an experimental `fetch` built-in instead of risking folks copy-pasting this? 🙏

Reply:
Agreed, let's hold off here.